app/pages/amigo-requests/amigo-requests.html

```diff
@@ -7,7 +7,9 @@
     <logon-required>
 
         <amigo-share-request-card *ngFor="let amigoShareRequest of amigoShareRequests"
-                                  [amigoShareRequest]="amigoShareRequest"></amigo-share-request-card>
+                                  [amigoShareRequest]="amigoShareRequest"
+                                  [accept]="accept"
+                                  [decline]="decline"></amigo-share-request-card>
```

Comment:
Passing functions into the component so that the page which has the entire list of amigo requests can remove them from the array.